src/ocean_data_assim/MOM_oda_driver.F90

```diff
@@ -198,8 +202,16 @@ subroutine init_oda(Time, G, GV, US, diag_CS, CS)
   call get_param(PF, mdl, "ASSIM_METHOD", assim_method,  &
        "String which determines the data assimilation method "//&
        "Valid methods are: \'EAKF\',\'OI\', and \'NO_ASSIM\'", default='NO_ASSIM')
-  call get_param(PF, mdl, "ASSIM_FREQUENCY", CS%assim_frequency,  &
-       "data assimilation frequency in hours")
+  call get_param(PF, mdl, "ASSIM_INTERVAL", CS%assim_interval,  &
+       "data assimilation update interval in hours",default=-1.0)
```

Comment:
We are about to require that all get_param calls for real variables have a units argument.  We are also trying to isolate all unit conversion factors in the scale arguments for get param calls.  It would be better to revise the call on lines 205 and 206 to read:

` call get_param(PF, mdl, "ASSIM_INTERVAL", CS%assim_interval,  &
       "data assimilation update interval in hours", &
       units=hours, default=-1.0, scale=3600.0*US%s_to_T)`

with a similar change to the get_param call for ASSIM_FREQUENCY on the next lines, and no need for the conversion on lines 213 below.  With this minor correction, this updated code will be in alignment with the changes to the get_param optional arguments that are about to happen, and it makes the code simpler and easier to read.